### ISSUE TITLE
Question about Caffe tutorial

### ISSUE THREAD

**Issue by waelHamdan** [BVLC/caffe#2125]
Hi all,
In the slide "Fine-tuning Tricks"
-where I can find "blobs_lr" in the Network, and how I can freeze it 
- for reducing learning rate, what is the minimal value I can use?

Thanks

**Comment by n-zhang** [comment]
The blobs_lr is defined in the training/test prototxt. You can set the blobs_lr to be 0 to freeze the back propagation on that specific layer. For learning rate, you can start from 1e-03 or 1e-4 and reduce by 10x.

We welcome you to look at the extensive documentation, tutorials, and examples at http://caffe.berkeleyvision.org/ 
However, please ask usage questions on [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) -- this issues tracker is primarily for Caffe development discussion. Thanks!
